src/main/java/study/carsharing/controller/MemberController.java

```diff
@@ -34,20 +36,16 @@ public String loginForm() {
         return "login";
```

Comment:
@kid7258 
This api is not valid.
Since you apply `@RestController` in this controller level, it will respond "login" as string.